Data/vocab.py

```diff
@@ -0,0 +1,46 @@
+#define Vocabulary class
```

Comment:
this is good thank you! i had a similar class called Vocabularies, but this is so much well written! Can you delete my vocabularies class in the preproc directory when this is merged? Thanks!

Reply:
Will do!